sdk/communication/azure-communication-rooms/README.md

```diff
@@ -0,0 +1,144 @@
+# Azure Communication Rooms client library for Python
+This package contains a Python SDK for Azure Communication Services for Rooms.
+Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview)
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please 
+refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install azure-communication-rooms
+```
+
+#### Prequisites
+
+- Python 3.6 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
+
+### Client Initialization
+
+To initialize the Rooms Client, the connection string can be used to instantiate.
+
+```python
+from azure.communication.rooms import RoomsClient
+from azure.identity import DefaultAzureCredential
+client = RoomsClient.from_connection_string(conn_str='<connection_str>' )
+```
+## Examples
+
+`Key parameters`
+
+- `valid_from`: A datetime object from which room will start existing
+- `valid_until`: A datetime object after which room meeting would end
+- `participants`: A list of RoomParticipant containing MRI's of invitees to the room
+All the above attributes are optional. The service provides default values of valid_until and
+valid_from if they are missing.
+
+`Create Room`
+```python
+from azure.communication.rooms import RoomsClient
+from azure.core.exceptions import HttpResponseError
+
+client = RoomsClient.from_connection_string(conn_str='<connection_str>')
+try:
+    response = client.create_room(
+        valid_from="start-datetime",
+        valid_until="meeting-end-datetime"
```

Comment:
These samples should use valid inputs, so I can copy, paste and run them.
These should be datetimes.